hoomd/md/pair/alch.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2009-2021 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
+# License.
+
+"""Alchemical potentials."""
+
+from hoomd.logging import log, Loggable
+from hoomd.operation import _HOOMDBaseObject
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeconverter import SetOnce
+from hoomd.md.pair import pair
+
+# alternative access? hoomd.alch.md.pair would be nice to somehow link here
+# without making overly complicated
+# I just like alch(MD/MC) as alchemy specification aliases
+
+# TODO: remove this variable when the log names are finalized
+_api = ['value', 'mass', 'mu', 'avalue', 'amomentum', 'net_aforce', 'aforces']
```

Comment:
I'm simply leaving this here because I'm already using the code and if I need to modify analysis code in the future, this will be useful.